mobile/src/navigation/app/components/drawer-content/drawer-content.tsx

```diff
@@ -1,20 +1,33 @@
-import React, { FC } from 'react';
-import { ScrollView, Button, Image } from 'react-native';
 import {
   DrawerContentComponentProps,
   DrawerItem,
 } from '@react-navigation/drawer';
-import { View, Text } from '~/components/common/common';
-import { AppScreenName, AppColor, AppIcon } from '~/common/enums/enums';
+import React, { FC } from 'react';
+import { Button, Image, ScrollView } from 'react-native';
+import { SvgXml } from 'react-native-svg';
+
+import Billing from '~/assets/icons/billing.svg';
+import Book from '~/assets/icons/book.svg';
+import Education from '~/assets/icons/education.svg';
+import Home from '~/assets/icons/home.svg';
+import Mentors from '~/assets/icons/mentors.svg';
+import Settings from '~/assets/icons/settings.svg';
+import { AppColor, AppScreenName } from '~/common/enums/enums';
+import { Text, View } from '~/components/common/common';
+
 import { styles } from './styles';
 
 const CustomDrawerContent: FC<DrawerContentComponentProps> = (props) => {
```

Comment:
Use destructuring approach for props to inject navigate callback, state with index and routes